_contentTemplates/common/get-started.md

```diff
@@ -13,59 +13,58 @@ The latest version of Telerik UI for Blazor is `{{site.uiForBlazorLatestVersion}
 #add-nuget-feed
 ### Add the Telerik NuGet Feed to Visual Studio
```

Comment:
This section addresses the needs of the trial users but not of the users with commercial licenses. Should we add something for the users with license here?

Reply:
The two tutorial-like articles target trial users for the most part. This is designed so by management (hence the "download the components" section too).